frontend/src/Onboarding/RedirectScreen.tsx

```diff
@@ -55,14 +55,11 @@ interface Props {
 
 export const RedirectScreen: React.FC<Props> = ({ redirectUrl }) => {
   const dispatch = useDispatch();
-  const { academicYear, graduationYear, isAdvisor } = useSelector(
-    (state: AppState) => ({
-      academicYear: getAcademicYearFromState(state),
-      graduationYear: getGraduationYearFromState(state),
-      isAdvisor: getIsAdvisorFromState(state),
-    })
-  );
```

Comment:
Getting these values here would error, as there is no user in our redux store at this point in the flow